src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/M2ArgumentCollection.java

```diff
@@ -62,54 +65,72 @@
 
     /**
      * Minimum number of variant reads in pileup to be considered an active region.
+     * In the hypothetical case of extreme high quality alignments, consider adjusting parameter down to 1.
+     * In the rare case of extreme deep coverage and where low fraction alleles are not of interest, adjust up to 3.
+     * Providing genomic intervals of interest with -L, setting the tumor standard deviation to zero and
+     * setting minimum variants in pileup to zero forces the tool to consider all provided regions as active.
      */
-    @Argument(fullName = "min_variants_in_pileup", optional = true, doc = "Minimum number of reads in pileup to be considered active region.")
+    @Argument(fullName = "min_variants_in_pileup", optional = true, doc = "Minimum number of variant reads in pileup to be considered active region.")
     public int minVariantsInPileup = 2;
 
     /**
-     * How many standard deviations above the expected number of variant reads due to error we require for a tumor pielup to be considered active
+     * How many standard deviations above the expected number of variant reads due to error we require for tool to consider a tumor pileup active.
+     * Argument sets the z-score. Here, base qualities inform expected error rate.
+     * Providing genomic intervals of interest with -L, setting the tumor standard deviation to zero and
+     * setting minimum variants in pileup to zero forces the tool to consider all provided regions as active.
      */
-    @Argument(fullName = "tumorStandardDeviationsThreshold", optional = true, doc = "How many standard deviations above the expected number of variant reads due to error we require for a tumor pielup to be considered active.")
+    @Argument(fullName = "tumorStandardDeviationsThreshold", optional = true, doc = "How many standard deviations above the expected number of variant reads due to error we require for a tumor pileup to be considered active.")
     public int tumorStandardDeviationsThreshold = 2;
 
     /**
-     * Minimum allele fraction of variant reads in normal for a pileup to be considered inactive
+     * Minimum fraction of variant reads in normal for a pileup to be considered inactive. Applies to normal data in a tumor with matched normal analysis.
+     * For value of 0.1, at least one tenth of pileup must be variant for tool to consider it a germline variant site and therefore not a region of interest
+     * in the somatic analysis.
      */
-    @Argument(fullName = "minNormalVariantFraction", optional = true, doc = "Minimum number of reads in pileup to be considered active region.")
+    @Argument(fullName = "minNormalVariantFraction", optional = true, doc = "Minimum fraction of variant reads in normal pileup to be considered germline.")
```

Comment:
Changed to "Minimum fraction of variant reads in normal pileup to be considered a germline variant site and thus not of further interest."